service/cloudconfig/cloud_config.go

```diff
@@ -0,0 +1,33 @@
+package cloudconfig
+
+import (
+	k8scloudconfig "github.com/giantswarm/k8scloudconfig"
+)
+
+type MasterExtension struct{}
+
+func (me *MasterExtension) Files() ([]k8scloudconfig.FileAsset, error) {
+	return nil, nil
+}
+
+func (me *MasterExtension) Units() ([]k8scloudconfig.UnitAsset, error) {
```

Comment:
A custom unit will be needed to download the certs from Key Vault. This will come later.